sklearn/ensemble/_forest.py

```diff
@@ -1672,6 +1674,8 @@ class RandomForestRegressor(ForestRegressor):
     sklearn.tree.DecisionTreeRegressor : A decision tree regressor.
     sklearn.ensemble.ExtraTreesRegressor : Ensemble of extremely randomized
         tree regressors.
+    sklearn.ensemble.HistGradientBoostingRegressor : Gradient boosting that
+        is a non-parametric model accepting monotonicity constraints.
```

Comment:
How about something like the below? I like how the decision tree and extratrees entries are short/give basically a longer version of the class name. For HGBT I also added the "fast for large datasets" because I think that is a catchy bit of information.

```suggestion
    sklearn.ensemble.HistGradientBoostingRegressor : A Histogram-based Gradient Boosting Regression Tree, very fast for big datasets (n_samples >= 10 000).
```

(same comment applies to the classifier entry)